gcloud/doc.txt

```diff
@@ -0,0 +1,35 @@
+Steps to using the DAOS VOL connector in Google Cloud:
+
+Move our blueprint ```dvc-build.yaml``` into Google Cloud Editor.
```

Comment:
Looks like this doc could use some updates since it looks like the yaml file's changed names since this was written.